docs/.ember-cli

```diff
@@ -5,5 +5,6 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false
+  "disableAnalytics": false,
+  "port": "7700" 
```

Comment:
This is so I can run the `docs` and the `tests` at the same time. Before they were running on the same port.